web/html-forms.md

```diff
@@ -214,7 +214,11 @@ Only basic knowledge of HTML is required to create a form in HTML. You would nee
 ### Independent Practice
 Spend 15 minutes researching these topics online:
 - Learn about what happens when an HTML form is submitted. 
+    - [Sending Form Data By Mozilla](https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data)
+    - [HTML Forms By w3schools](https://www.w3schools.com/html/html_forms.asp)
 - Are there echo/debug servers that can print out what is submitted to the server?
+    - [PHP Form Handling](https://www.w3schools.com/php/php_forms.asp)
+    - [PHP Debugging Tips](https://stackify.com/php-debugging-tips/)
```

Comment:
Please remove these links as they are not relevant (we do not cover PHP in this curriculum)